### PR TITLE
Common container component

### PR DESCRIPTION
**Pull request by starjardin**
Hey team,

I have created a container for the common  container component. Please have a look at if you are available. Comments are highly appreciated.

Refs: 45e5wj

Thanks,
Honey!

### CODE REVIEW

File: src/components/CommonContainerComponent/CommonContainerComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { Header } from "../Header"
```
Comment (andrzej-onja) [inline src/components/CommonContainerComponent/CommonContainerComponent.tsx:1]: I think its better to rename it to CommonPageConteiner

**Review by andrzej-onja — COMMENTED** [review]
Well done! I like that we can see example placeholders to to show off working feature.

**Review by vahona — APPROVED** [review]

File: src/components/Header/index.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { FC } from 'react'
 import { Logo } from '../Logo/Logo'
 import styled from 'styled-components'
 import logo from '../Logo/images/vamalaka.svg'
-import searchIcon from '../../assests/searchIcon.svg'
+import searchIcon from '../../assets/searchIcon.svg'
```
Comment (Fodilahy-mena) [inline src/components/Header/index.tsx:5]: I am not sure about this but is it fine to use camel case with svg file like this or we can name it as we name our component? I mean, should it be `searchIcon` or `SearchIcon`?

File: src/App.tsx [hunk 3]
```diff
@@ -1,42 +1,33 @@
...
-                username: {ad.username} , title: {ad.title}
-              </div>
-            )
-          })}
-        </div>
-      </header> */}
-      <OnboardingPage/>
-    </div>
+    <Switch>
+      <Route exact path='/' component={OnboardingSelection} />
```
Comment (Fodilahy-mena) [inline src/App.tsx:25]: We should use `OnboardingPage` here, not `OnboardingSelection`.

**Review by Fodilahy-mena — APPROVED** [review]

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**